src/components/BalanceSummary/index.js

```diff
@@ -59,6 +60,9 @@ const BalanceSummary = ({
                   height={250}
                   currency={counterValue}
                   tickXScale={selectedTime}
+                  renderTickY={(val, account) =>
+                    account ? formatCurrencyUnit(account.unit, val) : val
```

Comment:
it should not be `val` for the else case, it should be `formatCurrencyUnit(counterValue.units[0], val)`

Reply:
ok I'll fix that